.gitmodules

```diff
@@ -10,6 +10,7 @@
 [submodule "playbooks/roles/pmbauer.tarsnap"]
 	path = playbooks/roles/pmbauer.tarsnap
 	url = https://github.com/open-craft/ansible-tarsnap
+	branch = master
```

Comment:
We made a fork recently to make a change needed to upgrade the tarsnap version. The upstream repo's last commit is December '16 so I'm not sure if it's worth trying to upstream that change.

Reply:
Many external roles simply do not seem worth it for us, and I guess this is one of them.  External roles are useful to quickly get started, and sometimes to do significant amounts of work for us, but once the overhead of configuring and adapting them outweighs their benefits we should move to our own roles, possibly based on the external role.  Installing tarsnap is really easy, and I think we should get rid of this external role in the long run.